integration_tests/run_pyspark_from_build.sh

```diff
@@ -253,7 +260,8 @@ else
     else
         # We set the GPU memory size to be a constant value even if only running with a parallelism of 1
         # because it helps us have consistent test runs.
-        exec "$SPARK_HOME"/bin/spark-submit --jars "${ALL_JARS// /,}" \
+        # `spark.jars` is the same as `--jars`, e.g.: --jars a.jar,b.jar...
+        exec "$SPARK_HOME"/bin/spark-submit --conf spark.jars=${PYSP_TEST_spark_jars} \
```

Comment:
if the above is acceptable here we can do
```suggestion
        if [[ -n "$PYSP_TEST_spark_jars" ]]; then
           jarOpts=(--conf spark.jars="$PYSP_TEST_spark_jars")
        elif [[ -n "$PYSP_TEST_spark_driver_extraClassPath" ]]; then
            jarOpts=(--driver-class-path "$PYSP_TEST_spark_driver_extraClassPath")
        fi
        # `spark.jars` is the same as `--jars`, e.g.: --jars a.jar,b.jar...
        exec "$SPARK_HOME"/bin/spark-submit "${jarOpts[@]}" \
```